tests/cases/distributed/create/partition.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE my_table (
+  a INT PRIMARY KEY,
+  b STRING,
+  ts TIMESTAMP TIME INDEX,
+)
+PARTITION BY RANGE COLUMNS (a) (
+  PARTITION p0 VALUES LESS THAN (10),
+  PARTITION p1 VALUES LESS THAN (20),
+  PARTITION p2 VALUES LESS THAN (MAXVALUE),
+);
+
+-- SQLNESS REPLACE (\d{13}) ID
+SELECT table_catalog, table_schema, table_name, partition_name, partition_expression, greptime_partition_id from information_schema.partitions WHERE table_name = 'my_table' ORDER BY table_catalog, table_schema, table_name, partition_name;
+
+-- SQLNESS REPLACE (\d{13}) REGION_ID
+-- SQLNESS REPLACE (\d{1}) PEER_ID
+SELECT region_id, peer_id, is_leader, status FROM information_schema.greptime_region_peers ORDER BY peer_id;
```

Comment:
why not select *

Reply:
Because the `peer_addr` is not consistent, I don't want to write another REPLACE command.